db/migrate/20150924122432_add_devise_to_users.rb

```diff
@@ -0,0 +1,48 @@
+class AddDeviseToUsers < ActiveRecord::Migration
+  def self.up
+    change_table(:users) do |t|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 't' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)